src/components/main-panel/main-panel.tsx

```diff
@@ -16,15 +16,16 @@ import {
     verticalListSortingStrategy,
 } from "@dnd-kit/sortable";
 import { ErrorDisplay } from "@lifesg/react-design-system/error-display";
-import { useState } from "react";
-import { TElement, useBuilder } from "src/context-providers";
+import { TElement, useBuilder, useDisplay } from "src/context-providers";
+import { DisplayToast } from "../common/toast/toast";
```

Comment:
toast should be barrel exported so it should be 
```
import { DisplayToast } from "../common";
```

Reply:
changes made in the last push